app/models/user/UserService.scala

```diff
@@ -249,12 +257,12 @@ class UserService @Inject()(conf: WkConf,
 
   def updateLastTaskTypeId(user: User, lastTaskTypeId: Option[String])(implicit ctx: DBAccessContext): Fox[Unit] =
     userDAO.updateLastTaskTypeId(user._id, lastTaskTypeId).map { result =>
-      userCache.remove(user._id)
+      removeUserFromCache(user._id)
       result
     }
 
   def retrieve(loginInfo: LoginInfo): Future[Option[User]] =
-    userDAO.findOne(ObjectId(loginInfo.providerKey))(GlobalAccessContext).futureBox.map(_.toOption)
+    findOneCached(ObjectId(loginInfo.providerKey))(GlobalAccessContext).futureBox.map(_.toOption)
```

Comment:
I noticed that this didn’t use the cache. I think it is safe for it to do so now. This method is used by silhouette when authenticating requests. This change may save us quite a few database queries